pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -504,7 +504,7 @@ private static void validateTierConfigList(@Nullable List<TierConfig> tierConfig
                 segmentSelectorType, tierName);
         Preconditions.checkState(TimeUtils.isPeriodValid(segmentAge),
             "segmentAge: %s must be a valid period string (eg. 30d, 24h) in tier: %s", segmentAge, tierName);
-      } else {
+      } else if (!segmentSelectorType.equalsIgnoreCase(TierFactory.FIXED_SEGMENT_SELECTOR_TYPE)) {
```

Comment:
anything to check for fixed_selector? maybe require a non-empty segment list?

Reply:
i left this one on purpose.. wanted users to have the flexibility to remove the list if needed, without having to remove the whole tierConfig